PIL/Image.py

```diff
@@ -588,11 +588,11 @@ def _dump(self, file=None, format=None):
 
         self.load()
         if not format or format == "PPM":
-            self.im.save_ppm(file)
+            self.im.save_ppm(file, **options)
```

Comment:
And looking at it, _dump is potentially used in a few places that aren't debugging related, and that helper is only compiled in when WITH_DEBUG is defined.